### ISSUE TITLE
Update specimen type guidance

### ISSUE THREAD

**Issue by lg-king** [CDCgov/prime-simplereport#5516]

**Comment by lg-king** [comment]
It looks like this is done already?

**Comment by lauraykerr** [comment]
@lg-king I had a draft, but my understanding was some guidance was still TBD based on what we decide to do about validation. Is that settled?

**Comment by lg-king** [comment]
Yeah actually, we're still working through what text values we're going to accept. The guidance you have in the draft is still accurate (they can copy the snomed code from the code lookup tool), but they should also be able to enter certain text values (we're just working through that value set now). I'll keep you posted!

**Comment by lauraykerr** [comment]
Checking in on this @lg-king — any updates about accepted values?

**Comment by lg-king** [comment]
Yes! We're planning on accepting any SNOMED code, text value in LIVD table that maps to SNOMED, and RS's value set of text values (which is slightly different than the "official names" in the LIVD table, but this includes what we've already been accepting). I think for simplicity we should just instruct users to grab the SNOMED from the code lookup (as you had suggested).
